tbx/project_styleguide/templates/patterns/molecules/partner_logos/partner_logos.html

```diff
@@ -3,7 +3,13 @@
 <div class="partners__logos {% if classes %}{{ classes }}{% endif %}">
     {% for partner_logo in partner_logos %}
         <div class="partners__logo-wrapper">
-            {% image partner_logo max-100x90 format-webp loading="lazy" class="partners__logo" %}
+            {% image partner_logo max-100x90 format-webp as partner_logo_image %}
```

Comment:
we probably do want `loading="lazy" for the partner logos streamfield block, as it might be below the fold. How much harm is there in having it above the fold on the home page?

Reply:
It's noted as a caution on the [web.dev article here](https://web.dev/articles/browser-level-image-lazy-loading#:~:text=Caution%3A%20Don%27t%20lazy%2Dload%20images%20that%20are%20likely%20to%20be%20in%2Dviewport%20when%20the%20page%20loads%2C%20especially%20LCP%20images.) but it doesn't say what the penalties are for doing so. 

I can add some logic in to add in the attribute for when it's used a streamfield vs when it's used on the homepage but i'll do it in another MR if that's ok to progress this one. I've got time today to do it.

Reply:
sure, thank you!